docs/monitoring/tag-support.md

```diff
@@ -2,109 +2,12 @@ title: Monitoring Tags
-| kubernetes.deployment.name | Kubernetes deployment name | 
-| kubernetes.namespace | Kubernetes namespace | 
-| kubernetes.pvc.name | Kubernetes Persistent Volume Clain name | 
-| kubernetes.pod.controlledby | Controller of the pod (deployment or daemonset or statefulset) |
-
-### Operating System Tags
 
-Below are the OS related tags sent as part of OS metrics
-
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| os.disk | Human readable name of the block device | 
-| os.disk.mountpoint | Mount point for the disk in the file system |
-| os.disk.fs.type | Type of the file system associated with the device |
-| os.network | Name of the network interface |
-
-### Process Tags
-
-Below are Process related tags sent as part of metrics/logs process info.
-
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| process.name | process/program name |
-| process.pid  | process identifier | 
-
-### Network Traffic Stats Tags
-
-Below are the tags sent as part of Network Traffic Status metrics:
-
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| network.topology.address | Local address of the network connection | 
-| network.topology.destination.address | Remote host address of the network connection | 
-| network.topology.destination.port | Remote port |
-| network.topology.protocol |  Protocol name (TCP or UDP) |
-| network.topology.outgoing | determines whether the connection is incoming (client connected to server) or outgoin(current machine connects to external server) |
-| network.topology.process.id | process's identifier that produces connection stats |
-| network.topology.process.name | name of the process's image that produces connection stats | 
-
-### Serverless Tags
-
-Below are Serverless related tags sent as part of metrics/logs in the Serverless environment.
-
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| function.name | Name of the Lamba function | 
-| function.version | Version of the Lamda function | 
-| function.request.id | Identifier of the invocation request | 
-
-
-### Other Special Tags
-
-Below are the tags that are reserved for future use.
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
 
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| os.host.ip | IP Address of the host/server | server.ip, server.address, host.ip, host.address, source.ip, source.address
-| service.name | Name of the service where the data is collected from |
-| service.id | Unique service identifier |
-| service.type | Service type e.g. `hadoop` |
-| span | Building block of trace in distributed tracing |
+All [Special Tags](../../special-tags) apply as well.
 
+The tags are automatically collected by agents while fetching metrics and sent as part of metrics to the receiver.
```

Comment:
I think not needed as we say that in line 4. We could mention that in case when 3rd party Agents are used, they should ensure tag names match those from our Common Special Tags.

Reply:
I'd just remove this altogether. Also, I'd not mention 3rd party Agents. I want to push only our Agents in our Docs.

Reply:
Ability to use 3rd party agents is an important selling point.

Reply:
Aha, okay. I'll add a mention of this.